recipe/meta.yaml

```diff
@@ -25,11 +25,16 @@ build:
   # Use a build number difference to ensure that the GPU
   # variant is slightly preferred by conda's solver, so that it's preferentially
   # installed where the platform supports it.
-  number: {{ build_number + 200 }}  # [(gpu_variant or "").startswith('cuda') and (gpu_variant != "cuda-11")]
-  number: {{ build_number + 100 }}  # [(gpu_variant == "cuda-11") or (gpu_variant == "metal")]
-  number: {{ build_number }}        # [gpu_variant == "none"]
-  string: cuda{{ cuda_compiler_version | replace('.', '') }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [(gpu_variant or "").startswith('cuda')]
-  string: cpu_{{ blas_impl }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                 # [gpu_variant == "none"]
+  number: {{ build_number + 250 }}  # [(gpu_variant or "").startswith('cuda') and (gpu_variant != "cuda-11") and (x86_64_opt == "v3")]
+  number: {{ build_number + 200 }}  # [(gpu_variant or "").startswith('cuda') and (gpu_variant != "cuda-11") and (x86_64_opt == "none")]
+  number: {{ build_number + 150 }}  # [(gpu_variant == "cuda-11") or (gpu_variant == "metal") and (x86_64_opt == "v3")]
```

Comment:
gpu_variant won't be metal for x86_64, that's an arm-only variant. I guess it should be
```
(gpu_variant == "metal") or ((gpu_variant == "cuda-11") and (x86_64_opt == "v3"))
```
?

Reply:
and same for the line below

Reply:
I updated the selectors. The end result should be the same though.